lib/utils.js

```diff
@@ -20,38 +19,53 @@ const XCTEST_LOG_FILES_PATTERNS = [
 ];
 const XCTEST_LOGS_CACHE_FOLDER_PREFIX = 'com.apple.dt.XCTest';
 
-function parseWebkitDebugProxyPort (driverArgs) {
-  if (_.isEmpty(driverArgs)) {
-    return WEBKIT_DEBUG_PROXY_PORT;
+const SERVER_ARGS = {
+  'webkitDebugProxyPort': {
```

Comment:
I would prefer the same notation that we use for desired capabiities. Then there will be no need in code duplication

Reply:
See https://github.com/appium/appium-base-driver/blob/de7e41bbf7e7dc9c61749f094b997f95e73eef9b/lib/basedriver/capabilities.js#L24